### DISCUSSION TITLE
Rails Guides chapter overflow on mobile

### DISCUSSION THREAD

**Discussion by mwala-zm** [rails/rails#53322] [Docs]
![Screenshot_20240930-212250_Chrome](https://github.com/user-attachments/assets/56503b2d-aa4b-40fe-8e75-79a0c2604e52)

I noticed that the **Testing Rails Applications** chapter of the rails guides was overflowing on the x-axis on mobile. I'll update this issue if more pages are affected too.

@AmandaPerino

**Comment by khall** [comment]
The Assertion/Purpose table is responsible. Other pages have tables that produce similar results.

**Comment by JuanVqz** [comment]
@mwala-zm
Can this PR https://github.com/rails/rails/pull/54590 be a potential solution? 
Could you please report back if this is fixing it? https://044c0c42.rails-docs-preview.pages.dev/guides/debugging_rails_applications

Can you please share the url?
